### DISCUSSION TITLE
Horizontal gesture for in-workspace scrolling

### DISCUSSION THREAD

**Discussion by vollowx** [YaLTeR/niri#203] [Ideas]
It's strange that a _scrollable_ workspaces is not as it feels, and sometimes I just want to view the windows with only one hand using the touchpad.

Would be great if this'll implemented!

_Though this looks like a hard-to-impl feature..._

**Comment by YaLTeR** [comment]
Yeah I've already got an issue here: https://github.com/YaLTeR/niri/issues/100

**Reply by vollowx** [reply]
Oh I didn't notice that, really sorry for this!
